aten/src/ATen/native/ForeachUtils.h

```diff
@@ -87,6 +87,11 @@ bool check_fast_path_restrictions(
       }
     }
 
+    // FIXME(crcrpar) This is a workaround for BinaryOpScalarList which I see numerical errors.
```

Comment:
hm this is really weird

Reply:
The cause seemed to be my wrong specialization, fixed in:  https://github.com/pytorch/pytorch/pull/101760/commits/aa62e0579cbabe9ba19c7b927cacbdf5118b978b